src/pages/index.js

```diff
@@ -202,3 +193,154 @@ function closeModalOnEsc(evt) {
 }
 
 enableValidation(settings);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "1c9b6b75-fdbb-45e6-b261-c4a3ce5c7c23",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getUserInfo()
+  .then((userData) => {
+    profileName.textContent = userData.name;
+    profileDescription.textContent = userData.about;
+    profileAvatar.src = userData.avatar;
+    userId = userData._id;
+
+    return api.getInitialCards();
+  })
+  .then((initialCards) => {
+    initialCards.forEach((card) => {
+      const cardElement = getCardElement(card);
+      galleryList.prepend(cardElement);
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+// -----Submit New Post to Gallery-----
+
+function handleNewPostFormSubmit(evt) {
+  evt.preventDefault();
+  const originalNewPostSubmitButton = newPostSubmitButton.textContent;
+  const inputValues = {
+    name: inputCaption.value,
+    link: inputImageLink.value,
+  };
+  newPostSubmitButton.textContent = "Saving...";
+  api
+    .addCard(inputValues)
+    .then((newCardData) => {
+      const newCard = getCardElement(newCardData);
+      return galleryList.prepend(newCard);
+    })
+    .then(() => {
+      closeModal(newPostModal);
+      newPostSubmitButton.textContent = originalNewPostSubmitButton;
```

Comment:
`      newPostSubmitButton.textContent = originalNewPostSubmitButton;
`

It's not needed in `.then` , because you have it in `.finally`